src/js/App/Header/UserToggle.js

```diff
@@ -20,9 +23,14 @@ function buildItems(username, isOrgAdmin, accountNumber = -1, isInternal, extraI
     </DropdownItem>,
     <React.Fragment key="account wrapper">
       {accountNumber > -1 && (
-        <DropdownItem key="Account" isDisabled>
+        <DropdownItem key="Account" isPlainText>
           <dl className="chr-c-dropdown-item__stack">
-            <dt className="chr-c-dropdown-item__stack--header">Account number:</dt>
+            <dt className="chr-c-dropdown-item__stack--header">
+              Account number:
+              <Tooltip id="accountNumber-tooltip" content={accountNumberTooltip}>
```

Comment:
I think there should be some spacing between the text and the icon. Probably `pf-u-ml-sm` class on the tooltip.

Reply:
the `pf-u-ml-sm` wasn't working but I've added some spacing between the icon and title

Reply:
@InsaneZein ml is marging left. There are equivalent classes for other directions. If that does not work either, you can use the PF CSS variables: https://www.patternfly.org/v4/utilities/spacing

Reply:
:sweat_smile: the styling worked this time. I don't why it didn't last time. 

![image](https://user-images.githubusercontent.com/16109803/183086211-3c1cb8ea-baea-4a87-a8d0-b5e7d2b436b6.png)